diagnostic.sql

```diff
@@ -3,26 +3,44 @@ CREATE DATABASE "sql-diagnostic";
 \pset pager 0
 \c "sql-diagnostic"
 
--- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
-
--- your code here
-
--- Question 2. Add `Bishop, Caren, f, 63, 132, 1943-09-26, Brown, Black` into the table of licensees.
+-- Question 1. Create a table called `licensees` to hold data in columns surname,
+--given_name, gender, height, weight, born_on, eye_color, hair_color.
+--See `data/licensees.csv` for example data.  Don't forget that all tables
+-- should have an id column as a unique identifier for a row.
+
+CREATE TABLE licensees (
+   surname varchar,
```

Comment:
Don't forget the `id serial PRIMARY KEY` part. Even though the database will create an ID for you automatically, you still need to tell it to do so